src/app/app.component.html

```diff
@@ -6,40 +6,45 @@ <h2>Unsupported Browser</h2>
     </div>
   </div>
 }
-@if (currentUrlWithoutHash$ | async; as currentUrl) {
-  <div class="skip-to-content-link">
-    <!--
-    Can't use a simple relative link for the URL because we have base href set of /d/ set in non-dev environments
-    which is needed for links to assets but would break this.
-    -->
-    <a [href]="currentUrl.toString() + '#content'">Skip to content</a>
-  </div>
-}
 
-@if (isDataLoaded) {
-  <biggive-main-menu
-    [donateUrlPrefix]="donateUriPrefix"
-    [blogUrlPrefix]="blogUriPrefix"
-    [experienceUrlPrefix]="experienceUriPrefix"
-    [isLoggedIn]="isLoggedIn"
-    [smallCharityWeekEnabled]="true"
-    [someCampaignHasHomePageRedirect]="someCampaignHasHomePageRedirect()"
-  ></biggive-main-menu>
-}
+<header aria-label="Menu &amp; skip">
```

Comment:
I think the label is probably superfluous here but not a blocker

Reply:
Yep agreed, I was testing this because I got worried something about the container was affecting the skip link's keyboard focusability. Seems like I did something wrong or it was an unrelated WIP change.